karma.conf.js

```diff
@@ -0,0 +1,198 @@
+
+      'js/src/chrome/projects/chrome.projects.js',
+      'js/src/chrome/users/chrome.users.js',
+      'js/src/chrome/settings/chrome.settings.js',
+      'js/src/chrome/settings/chrome.settings.filters.js',
+      'js/src/chrome/settings/chrome.settings.cartocss.js',
+      'js/src/chrome/settings/chrome.settings.tooltip.js',
+      'js/src/chrome/settings/chrome.settings.settingsselector.js',
+      'js/src/chrome/settings/chrome.settings.styler.js',
+      'js/src/chrome/settings/styler.js',
+      'js/src/chrome/settings/styler.polygon.js',
+      'js/src/chrome/settings/styler.point.js',
+      'js/src/chrome/settings/styler.line.js',
+      'js/src/chrome/settings/styler.legend.js',
+      'js/src/chrome/settings/styler.raster.js',
+      'js/src/chrome/settings/chrome.settings.layers.js',
+      'js/src/chrome/settings/chrome.settings.extras.js',
+
+
+      // Controls 
+      'js/src/controls/control.js',
+      'js/src/controls/control.zoom.js',
+      'js/src/controls/control.geojson.draw.js',
+      'js/src/controls/control.draw.js',
+      'js/src/controls/control.zindex.js',
+      'js/src/controls/control.measure.js',
+      'js/src/controls/control.geolocation.js',
+      'js/src/controls/control.layermenu.js',
+      'js/src/controls/control.description.js',
+      'js/src/controls/control.mouseposition.js',
+      'js/src/controls/control.baselayertoggle.js',
+      'js/src/controls/control.style.js',
+      'js/src/controls/control.tooltip.js',                
+      'js/src/controls/control.spinningmap.js',
+      'js/src/controls/control.graph.js',
+      'js/src/controls/control.graph.snow.js',
+      'js/src/controls/control.animator.js',
+      'js/src/ext/popup.chart.js',
+      'js/src/controls/control.chart.js',
+      'js/src/controls/control.wms.js',
+      
+
+      // Models 
+      'js/src/models/model.js',
+      'js/src/models/model.project.js',
+      'js/src/models/model.user.js',
+      'js/src/models/model.layer.js',
+      'js/src/models/model.layer.cube.js',
+      'js/src/models/model.layer.vector.js',
+      'js/src/models/model.layer.raster.js',
+      'js/src/models/model.layer.providers.js',
+      'js/src/models/model.layer.topojson.js',
+      'js/src/models/model.layer.geojson.js',
+      'js/src/models/model.layer.wms.js',
+      'js/src/models/model.file.js',
+
+      // Analytics
+      'js/src/ext/analytics.js',
+
+      // Satellite angle
+      'js/src/ext/satelliteAngle.js',  
+
+      // Buttons
+      'js/src/ext/buttons.js',
+
+      // Buttons
+      'js/src/ext/dropdown.js',
+
+      // Language file
+      'js/src/lang/language.english.js',
+
+      // Extend Leaflet
+      'js/src/ext/extendLeaflet.js',
+      
+      // Momory
+      'js/src/tests/memory.js',
+      
+      // App 
+      'js/src/core/app.js',
+
+      'test/config.mock.js',
+
+      'test/**/*.spec.js'
+
+    ],
+
+    port: 3001,
+    
+
+    reporters: ['progress'],
+
+    browsers: ['Chrome']
```

Comment:
<img width="668" alt="screen shot 2016-11-07 at 20 40 41" src="https://cloud.githubusercontent.com/assets/2197944/20072884/7c5cc32e-a52a-11e6-8db9-57591b4085a2.png">

I'm getting this on OSX and Ubuntu. I think we should use [PhantomJS](https://github.com/karma-runner/karma-phantomjs-launcher) instead, as it's the browser we're already using on Engine.